pypeit/specobj.py

```diff
@@ -211,10 +211,8 @@ def _init_internals(self):
         # Object finding
         self.smash_peakflux = None
         self.smash_nsig = None
-        self.maskwidth = None
 
         # Hand
-        self.hand_extract_flag = False
```

Comment:
I think this should stay, otherwise for non manual extracted objects, this flag would be `None` instead of `False`. Is it correct?

Reply:
good catch, @deborape -- I thought it was just set to `None`
Please put it back and sorry about that.

Reply:
Ok. No problem. I'll do that.